tests/arch/arm/arm_interrupt/src/arm_interrupt.c

```diff
@@ -101,7 +101,8 @@ void test_arm_interrupt(void)
 	/* Verify that the spurious ISR has led to the fault and the
 	 * expected reason variable is reset.
 	 */
-	zassert_true(expected_reason == -1,
+	reason = expected_reason;
+	zassert_true(reason == -1,
```

Comment:
Maybe `zassert_equal(reason, -1, ...)` could be a better choice?

Reply:
Let's change it to that then :)
Done